cmd/things/main.go

```diff
@@ -53,6 +56,7 @@ const (
 	envDBUser    = "MF_THINGS_DB_USER"
 	envDBPass    = "MF_THINGS_DB_PASS"
 	envDBName    = "MF_THINGS_DB"
+	envCACerts   = "MF_THINGS_CA_CERTS"
```

Comment:
Can you, please, update the docs as well (README.md)?

Reply:
I think I've covered where this makes sense - should I leave the docker-compose with the default (blank) or do you want to include a sample CA along with your existing self-signed certs?

Reply:
@nwest1 you can reuse the certs and ca from [here](https://github.com/mainflux/mainflux/tree/master/docker/ssl). We use them for nginx for our dev environment

Reply:
I updated the getting started guide with some simple explanation. Unless you'd like me to configure nginx for grpc load balancing, I'm not sure it makes sense to secure this connection.

Another option would be to add a separate ENV var to control the `grpc.WithInsecure()` rather than the current logic. That pattern might scale out better than controlling two options with one parameter.

Reply:
@nwest1 it looks good, we can change that in future if there is a need it and if we notice some problems.

I would ask you one more thing and then we are done regarding the docs :)
Please add a note in the getting started guide that if the env vars for setting the ca are omitted, the gRPC communication will not be encrypted.

Reply:
Added :) Let me know if there are any other details!